src/app/admin/page.tsx

```
@@ -1,18 +1,24 @@
   description: "Welcome back to your admin dashboard",
 };
 
+export default async function AdminWelcome() {
+  const { metadata } = await getSessionData();
+  
+  if (!metadata?.data) {
+    return redirect('/auth/sign-in');
+  }
+  
+  const currentTime = new Date().toLocaleTimeString([], { hour: "2-digit", minute: "2-digit" });
+};
```

ESLint

[error] Multiple default exports.

(import/export)